aws/resource_aws_appconfig_environment.go

```diff
@@ -0,0 +1,240 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/service/appconfig"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags"
+)
+
+func resourceAwsAppconfigEnvironment() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsAppconfigEnvironmentCreate,
+		Read:   resourceAwsAppconfigEnvironmentRead,
+		Update: resourceAwsAppconfigEnvironmentUpdate,
+		Delete: resourceAwsAppconfigEnvironmentDelete,
+
+		Schema: map[string]*schema.Schema{
+			"application_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ValidateFunc: validation.All(
+					validation.StringLenBetween(1, 64),
+				),
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ValidateFunc: validation.All(
+					validation.StringLenBetween(0, 1024),
+				),
+			},
+			"monitor": {
```

Comment:
I think in the description this is: "monitors" plural, not singular

Reply:
See https://github.com/figma/terraform-provider-aws/pull/79#discussion_r594688082